cms/djangoapps/contentstore/tests/utils.py

```diff
@@ -11,7 +11,7 @@
 from opaque_keys.edx.keys import CourseKey
 from opaque_keys.edx.locations import AssetLocation
 
-from cms.djangoapps.contentstore.utils import reverse_url
+from contentstore.utils import reverse_url
```

Comment:
I think we added this during the `bulkupdate` work.
It should no longer be needed, now that dependencies have been cleaned up.